src/manifolds/PowerManifold.jl

```diff
@@ -151,18 +151,12 @@ The array `bases` stores bases corresponding to particular parts of the manifold
 
 The type parameter `F` denotes the [`AbstractNumbers`](@ref) that will be used as scalars.
 """
-struct PrecomputedPowerOrthonormalBasis{
-    TB<:AbstractArray{<:AbstractPrecomputedOrthonormalBasis},
-    F,
-} <: AbstractPrecomputedOrthonormalBasis{F}
+struct PowerBasisData{TB<:AbstractArray{T} where {T}}
```

Comment:
```suggestion
struct PowerBasisData{TB<:AbstractArray}
```
Both versions are equivalent here but this one is more clear.